yarn-project/aztec-rpc/src/aztec_rpc_server/aztec_rpc_server.ts

```diff
@@ -370,12 +370,12 @@ export class AztecRPCServer implements AztecRPC {
   ): Promise<ExecutionRequest> {
     const contract = await this.db.getContract(to);
     if (!contract) {
-      throw new Error('Unknown contract.');
+      throw new Error(`Unknown contract ${to}! Add it to Aztec RPC server by calling server.addContracts(...)`);
```

Comment:
```suggestion
      throw new Error(`Unknown contract ${to}: add it to Aztec RPC server by calling server.addContracts(...)`);
```
It's a matter of personal style, but I'd avoid using exclamations in error messages. Feel free to ignore.

Reply:
I have no problem with that 👍